### DISCUSSION TITLE
老大,有些话想对你说...

### DISCUSSION THREAD

**Discussion by SoyKoa** [shuax/MouseInc.Settings#462] [General]
用MouseInc很多年了,当听到他们说的时候我还有点不相信,后来一个水友发了知乎的链接看完了整个内容,知道事情的始末,感觉你好无辜,那4个小伙伴好幸运!
记得刚开始用手势软件的时候还加了群,当时也是从小白慢慢研究怎么用,后来还经常在群里回答他们的不懂的地方,然后突然有天群解散了,现在才明白原因,算是后知后觉吧,
之所以你成了最倒霉的那个,我个人看法是因为去了那地方,是个人都会心慌,六神无主,一进去就觉得自己成了有罪之人,如果能冷静一点,不去承认,最后你也不会有这样的遭遇,就不该认!!!他们是为了交差,而你成了壮丁.
做为一个网民,关心自己的隐私有没有被泄露,下载来看看,就有罪吗?发链接方便大家去查看自己的隐私有没有被泄露,就有罪吗?那有罪的人就太多了,自己本来就是不明真相的吃瓜群众,最后成了受害者,原来善良也是一种罪!人善被人欺!!
MouseInc的手册链接还保留着,说明你对软件还是放不下心的,至少有这份牵挂,遇到这些糟心的事情确实需要花时间来慢慢磨平,但它是缓的,不用进去,愿你能早日走出来!

**Comment by Ctrl-Ling** [comment]
同愿早日走出来

**Comment by sumeco** [comment]
确实，我是律师，其实他这事情在侦查阶段委派律师可以做到无罪辩护。

**Reply by shuax** [reply]
刑事律师吗？我需要帮助

**Reply by lingshow1** [reply]
我们是大成Dentons成都办公室刑事组的律师，我本人也是mouseinc的用户。如果需要帮助，请联系我。
myron.yu@outlook.com

**Reply by shuax** [reply]
> 我们是大成Dentons成都办公室刑事组的律师，我本人也是mouseinc的用户。如果需要帮助，请联系我。 [myron.yu@outlook.com](mailto:myron.yu@outlook.com)

发了邮件没反应，是不是到垃圾箱了……

**Comment by sanorintea** [comment]
加油！

**Comment by CaiSijian** [comment]
https://flk.npc.gov.cn/
作者大大，这个是国家法律数据库的链接，我不是很懂法律，但是希望能帮到你
望作者能得到轻判或者无罪辩护

**Comment by lingshow1** [comment]
今天在忙，收到了。

大概看了一下你的情况，请问有什么要询问的。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by CaiSijian** [comment]
你好，你可能误会了，我也是mouseinc的用户，觉得作者写的软件很好，故此留言留下国家法律数据库的地址




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "shuax/MouseInc.Settings"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月1日(星期二) 下午5:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shuax/MouseInc.Settings] 老大,有些话想对你说... (Discussion #462)





 今天在忙，收到了。 
 
 大概看了一下你的情况，请问有什么要询问的。 
 
 获取Outlook for Android<https://aka.ms/AAb9ysg&gt; 
 ________________________________
